### DISCUSSION TITLE
Do I need a Garmin account to use this?

### DISCUSSION THREAD

**Discussion by ghost** [tcgoetz/GarminDB#239] [General]
My goal is to use the watch without a Garmin account and without the Garmin Connect app as I find mobile phones too small at my age.
I would like to be able to connect the watch to my Windows PC, download the data and analyze it on my big PC screen.
Is this possible using this app?

**Comment by jebins** [comment]
Hi,

Same question here: I would like to use my Garmin Watch without a Garmin Connect account.

The description of this repository states that garmindb can :

> Download and parse data from Garmin Connect **or a Garmin watch**

But I struggle to find documentation related to this use-case.

**Comment by lpierron** [comment]
1. Use Garmin Express (https://www.garmin.com/en-US/software/express/mac/) to connect and mount your Garmin Watch on your system.
2. Check the entry `copy:mount_dir` or `directories:mount_dir` in your configuration file: `GarminConnectConfig.json`. Be sure that your watch content is mounted on the right `directory`: `mount_dir`.
3. Copy the datas in your GarminDB folder with: `garmindb_cli.py -c`

**Comment by ghost** [comment]


**Reply by lpierron** [reply]
Garmin Express need an account on Garmin Connect Web, with your watches identified. Garmin Express *mounts* your watch on your PC File System and after that you can use `garmindb_cli.py -c` to copy the files from your watch volume to your GarminDB database.
Maybe you can find some information, on how to *mount* Garmin watches without Garmin Express: https://www.reddit.com/r/running/comments/rcg9yl/garmin_watches_and_computers_running_linux_eg/
